chat-widget/src/components/loadingpanestateful/common/errorUIStyleProps/errorUILoadingPaneStyleProps.ts

```diff
@@ -0,0 +1,52 @@
+import { ImageFit } from "@fluentui/react";
+import { ILoadingPaneStyleProps } from "@microsoft/omnichannel-chat-components/lib/types/components/loadingpane/interfaces/ILoadingPaneStyleProps";
+import { AlertIcon } from "../../../../assets/Icons";
+
+export const errorUILoadingPaneStyleProps: ILoadingPaneStyleProps = {
```

Comment:
not a biggie, but it seems like the pattern is the name of the resource file to be in capital